tabpy/tabpy_server/handlers/evaluation_plane_handler.py

```diff
@@ -44,6 +44,10 @@ def initialize(self, executor, app):
 
     @gen.coroutine
     def _post_impl(self):
+        if not self.eval_enabled:
+            self.error_out(400, "Evaluate action is disabled.")
```

Comment:
Minor point, but I think "endpoint" would be more clear than "action" since that is the term we use in the documentation.

Reply:
Or something like, add-hoc script is not allowed. Better confirm with Nathan about the string.

Reply:
Changed to "Evaluate endpoint is disabled" for now

Reply:
As per Nathan, changed to "Ad-hoc scripts have been disabled on this analytics extension, please contact your administrator."